docker/README.md

```diff
@@ -1,33 +1,49 @@
-# Radicale docker image
+# Radicale Docker image
 
-# Build the image
+This image neither features HTTPS or a webserver. It's recommended to use a 
+reverse proxy in front of it.
 
-1. Change into the root directory of the repo.
-1. `docker build -t radicale .`
+## Building the image
 
-# Running the image
+    cd docker
+    make
 
-```bash
-docker run --rm \
-    -v $PWD/lib:/var/lib/radicale \
-    -v $PWD/conf:/etc/radicale:ro \
-    radicale
-```
+## Running a container
 
-This image neither features HTTPS or a webserver. It's recommended to use a reverse proxy in front of it.
+Radicale can be run in an ephemeral container that mounts the folders `conf` 
+and `lib` from the current working directory:
 
-# Reference
+    docker run --rm \
+        -v $PWD/lib:/var/lib/radicale \
+        -v $PWD/conf:/etc/radicale \
+        kozea/radicale:<version>
 
-## Parameters
+There's a minimal, working example configuration for Docker Compose in the 
+`docker/compose` directory. Copy its contents to a location with a meaningful
+name (`radicale` should meet the criteria) change your working directory to it
+and run Radicale without attaching stdout:
+
+    docker-compose up -d
+
+## Reference
+
+### Parameters
 
 - `RADICALE_UID`: The user id of the internal radicale user
 - `RADICALE_UID`: The group id of the internal radicale group
 
-## Volumes
+### Volumes
 
 - `/var/lib/radicale`: data files
 - `/etc/radicale`: Radicale configuration (put the configuration file here)
 
-## Ports
+### Ports
+
+- `5323`: Radicale's http port
+
+### Creating a user in a running container
+
+    docker exec -ti <ontainer> htpasswd -B -C 16 /etc/radicale/users <username>
```

Comment:
You're running here as the root user and not the radicale user. You possibly have to restart the host or manually chown the file if it's fresh.

Reply:
d'oh. i had tested that against an existing file. i add the `--user` option to the example.